### PR TITLE
leaflet map

### PR DESCRIPTION
**Pull request by epartakki**
- Map is interactive and has pop-ups with the names of the stations
- The submodule I added is for Heroku, so you can ignore it for now

### CODE REVIEW

File: app/views/sessions/map.html.erb [hunk 1]
```diff
@@ -1,105 +1,46 @@
...
-
-//Creates the open street map tile layer
-  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+var map = L.map('map').setView([42.325, -72.65], 14);
+
+
+L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
     attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-}).addTo(map); //Adds to established map
+}).addTo(map);
```
Comment (sunniraleigh) [inline app/views/sessions/map.html.erb:18]: Would you like to add a loop for this so it's pulling station data from the db?

**Review by sunniraleigh — APPROVED** [review]
Amazing job!! Looks so good <3

**Review by luciebrock — APPROVED** [review]
epic sauce

**Review by kvangreen — APPROVED** [review]
Amazing work <3

**Review by amarrap — APPROVED** [review]
Yay! Cool!

**Pull request merged**